feature-libs/product/configurators/common/src/components/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts

```diff
@@ -16,6 +16,7 @@ import { Observable, of } from 'rxjs';
 import { take } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
 import { ConfigPreviousNextButtonsComponent } from './config-previous-next-buttons.component';
+import { ConfigUtilsService } from '@spartacus/product/configurators/common';
```

Comment:
Please consume via path here!

Reply:
You are absolutely right. I will adjust it